tests/smoke/setup_docker.sh

```diff
@@ -11,4 +11,6 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make setup-ui-tests
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml stop nginx
-docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d
\ No newline at end of file
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d
+# Pull and run selenium firefox docker image
+docker run -p 4444:4444 --detach selenium/standalone-firefox
```

Comment:
This works for me, as in, I can curl `localhost:4444` and see some output. 

However, I am a bit stuck in trying to test out the tests locally. I nuked all of my containers and images so I could start over. Now, I get: ERROR: pull access denied for addons-frontend, repository does not exist or may require 'docker login'. It does not build the image automatically.

I'm not really sure what's going on but I guess you might as well merge this so you can test on circle CI.